examples/OpenTracing.Examples/TestUtils.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using OpenTracing.Mock;
+using OpenTracing.Tag;
+using Xunit;
+
+namespace OpenTracing.Examples
+{
+    public static class TestUtils
+    {
+        public static TimeSpan DefaultTimeout { get; } = TimeSpan.FromSeconds(5);
+
+        public static void WaitForSpanCount (MockTracer tracer, int spanCount, TimeSpan timeout)
```

Comment:
I think, ideally, this method should not be necessary if everything is `await`ed properly.

Reply:
I left it as I still need to for the ClientServer example - specially because I'm using an actual `Thread` to act as the server ;) But the rest of the examples do not use it anymore.

Reply:
@carlosalberto did you forget to push your changes? I don't yet see them here.